### PR TITLE
improvements: mobile and desktop Header revamp

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
mobile and desktop Header revamp

### Types of changes
improvements

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards

### CODE REVIEW

**Review by imnavanath — APPROVED** [review]

File: inc/markup-extras.php [hunk 1]
```diff
@@ -326,10 +326,8 @@ function astra_get_site_title_tagline( $display_site_title, $display_site_taglin
 		if ( $display_site_title || $display_site_tagline ) {
 			/* translators: 1: Site Title Markup, 2: Site Tagline Markup */
 			$html .= sprintf(
-				'<div class="ast-site-title-wrap">
-						%1$s
-						%2$s
-					</div>',
+				'%1$s
+					%2$s',
```
Comment (premanshup) [inline inc/markup-extras.php:330]: @sagarb90 Please verify once this with the old header as well.

File: sass/site/header/_header-layouts.scss [hunk 2]
```diff
@@ -238,7 +238,7 @@
 		padding-right: 1em;
 	}
 	&.header-main-layout-2{
-		.site-logo-img + .ast-site-title-wrap .site-title{
+		.site-logo-img + .site-title{
```
Comment (premanshup) [inline sass/site/header/_header-layouts.scss:241]: @sagarb90 Verify with old Header and Addon options.

File: inc/builder/type/header/above-header/assets/js/customizer-preview.js [hunk 3]
```diff
@@ -78,6 +78,6 @@
 	astra_builder_advanced_css( 'section-above-header-builder', '.ast-above-header.ast-above-header-bar' );
 	
     // Advanced Visibility CSS Generation.
-	astra_builder_visibility_css( 'section-above-header-builder', '.ast-above-header-bar', 'block' );
+	astra_builder_visibility_css( 'section-above-header-builder', '.ast-above-header-bar', 'grid' );
```
Comment (premanshup) [inline inc/builder/type/header/above-header/assets/js/customizer-preview.js:81]: @sagarb90 Why are we changing the default value from Block to Grid? and does this require backward compatibility?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/builder/markup/class-astra-builder-footer.php [hunk 4]
```diff
@@ -223,14 +223,12 @@ public function footer_copyright() {
 
 			$content = astra_get_option( 'footer-copyright-editor' );
 			if ( $content || is_customize_preview() ) {
-				echo '<div class="ast-footer-copyright">';
-					echo '<div class="ast-footer-html-inner">';
+				echo '<div class="ast-footer-copyright ast-footer-html-inner">';
```
[the remark this answers is not in the document]
Comment (sagarb90) [reply inc/builder/markup/class-astra-builder-footer.php]: @premanshup Removed class ast-footer-html-inner from both theme and from add-on

File: inc/builder/type/header/above-header/assets/js/customizer-preview.js (hunk 3 above)
Comment (sagarb90) [reply inc/builder/type/header/above-header/assets/js/customizer-preview.js]: @premanshup Yes Reverted this change as we have already handle this CSS

File: sass/site/header/_header-layouts.scss (hunk 2 above)
Comment (sagarb90) [reply sass/site/header/_header-layouts.scss:241]: @premanshup Yes verified and working fine.

File: inc/markup-extras.php (hunk 1 above)
Comment (sagarb90) [reply inc/markup-extras.php:330]: @premanshup Yes verified and working fine as it replace the div with %1$s & %2$s

**Review by premanshup — APPROVED** [review]

**Pull request merged**